api/store/mongo/device_store.go

```diff
@@ -69,6 +71,46 @@ func (s *Store) DeviceList(ctx context.Context, pagination paginator.Query, filt
 		}}, query...)
 	}
 
+	// When status is "accepted", the user should not be able to accept the device again.
+	if status == "accepted" {
+		query = append(query, bson.M{
+			"$addFields": bson.M{
+				"acceptable": false,
+			},
+		})
+	}
+
+	// When status is "pending" or "rejected", the user should be able to accept the device only if the maximum number
+	// of devices has not been reached.
+	if status == "pending" || status == "rejected" {
+		if mode&store.DeviceListModeMaxDeviceReached != 0 {
```

Comment:
Is there a simple/better way to check if mode "is" a store.DeviceListModeMaxDeviceReached.